app/src/main/java/io/left/meshim/database/MeshIMDao.java

```diff
@@ -77,12 +80,13 @@
      *     direction because we will not have exchanged a message with every user in the
      *     Users table.
      * </p>
-     * @return a summary of every conversation the device's user has started
+     * @return a summary of every conversation the device's user has started, and number of unread messages
      */
-    @Query("SELECT Username, Avatar, Contents, Timestamp, PeerID "
+    @Query("SELECT Username, Avatar, Contents, Timestamp, PeerID, isRead, numberOfUnreadMessages "
```

Comment:
We use `PascalCase` for database columns.

Also a shorter name like even just `UnreadMessages` is probably fine.